### PR TITLE
Minicopter 

### PR DESCRIPTION
**Pull request by laikadoggirl**
Added a smaller, much weaker helicopter for fun
Seats two people, one gunner, with a bobby gun mounted on the front, and a driver

Takes fuel or oil

nugget did the sprite

### CODE REVIEW

**Comment by xcs011** [comment]
nugget did the sprite 

nugget stop complaining see I did it

**Comment by Jim-Smiff** [comment]
tested with no bugs 100% fine and fun

**Comment by xcs011** [comment]
we even tested this, are you proud of me vamist?

**Comment by xcs011** [comment]
Either I'm dumb or github is, one of us deleted the Readme by accident, I'll fix it tomorrow

**Comment by laikadoggirl** [comment]
I highly doubt you will deal with this tomorrow vamist, but don't, upon further reflection the PR needs a little more tweaking:

TODO:
~~make heli grapplable because its funny~~ no

add a wooden crate and gramophone attachment point

add fortunate son disc >:)

**Review by Vam-Jam — CHANGES_REQUESTED** [review]
Re-check what you've commited, you've removed zip, meteor shittery and build_release

**Comment by laikadoggirl** [comment]
Yeah meteor is still out so I can test, will be fixed.
I honestly don't know when build_release and all that got deleted, Readme vanished at one point too

**Comment by Vam-Jam** [comment]
> Yeah meteor is still out so I can test, will be fixed.
> I honestly don't know when build_release and all that got deleted, Readme vanished at one point too

You dont have to commit all edited files, git allows you to select what files you'd like to commit, the rest got deleted because KAG will remove files it cant send to clients (e.g. anything aside from .as, .ogg and so on)

**Comment by laikadoggirl** [comment]
dont worry about all the readme shit
god I fucking hate resolving issues

fixed all the other stupid shit though with kag deleting files

**Comment by laikadoggirl** [comment]
@Vam-Jam just merge it if its good, ill do an attachment update later for the crate and gramophone

**Comment by laikadoggirl** [comment]
@Vam-Jam test already nerd, also I finished letting noisemakers and gramophones attach and detach

**Review by laikadoggirl — COMMENTED** [review]
They done

**Comment by laikadoggirl** [comment]
I found the funny request review button also wrecks added

File: TFlippy_TerritoryControl_Weapons_Dev/Entities/Vehicles/Helichopper/Helichopper.as [hunk 1]
```diff
@@ -39,6 +39,7 @@ void onInit(CBlob@ this)
 
 	this.Tag("vehicle");
 	this.Tag("aerial");
+	//this.Tag("helicopter");    					remind laika later when someone finally sees this
```
Comment (Vam-Jam) [inline TFlippy_TerritoryControl_Weapons_Dev/Entities/Vehicles/Helichopper/Helichopper.as:42]: .

File: TFlippy_TerritoryControl_Patreon_Dev/Items/Noisemaker/Noisemaker.cfg [hunk 2]
```diff
@@ -70,7 +70,8 @@ $brain_factory                          =
 $attachment_factory                     = box2d_attachment
 @$attachment_scripts                    = 
 # name; pixel offset (from center) X; offset Y; socket/plug 0/1; controller; radius
-@$attachment_points                    = PICKUP; -3; -1; 1; 0; 0;	
+@$attachment_points                     = PICKUP; -3; -1; 1; 0; 0;
+										  CARGO; 0; -10; 1; 0; 0;
```
Comment (Vam-Jam) [inline TFlippy_TerritoryControl_Patreon_Dev/Items/Noisemaker/Noisemaker.cfg]: indents

File: TFlippy_TerritoryControl_Weapons_Dev/Entities/Vehicles/Helichopper/Helichopper.cfg [hunk 3]
```diff
@@ -63,6 +63,7 @@ $attachment_factory                               = box2d_attachment
 @$attachment_scripts                              = 
 @$attachment_points                               = DRIVER; 22; -3; 0; 0; 24;
                                                     PASSENGER; 15;-6;0;0;12;
+													CARGO; 0; 14;  0; 0; 0;
```
Comment (Vam-Jam) [inline TFlippy_TerritoryControl_Weapons_Dev/Entities/Vehicles/Helichopper/Helichopper.cfg]: indents

File: TFlippy_TerritoryControl_Weapons_Dev/Entities/Vehicles/Wreck/MinicopterWreck.cfg [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+@$attachment_scripts                       = 
+@$attachment_points                        = 						  					  
+
+$inventory_factory                         =
+
+$name                                      = minicopterwreck
+@$scripts                              	   = Wooden.as;
+											 WoodenHit.as;  
+											 Shop.as;
+											 Wreck.as;
```
Comment (Vam-Jam) [inline TFlippy_TerritoryControl_Weapons_Dev/Entities/Vehicles/Wreck/MinicopterWreck.cfg]: indents

**Review by Vam-Jam — CHANGES_REQUESTED** [review]

**Pull request merged**